CollapsibleExample/MHKeyValuePair.h

```diff
@@ -0,0 +1,25 @@
+//
+//  MHKeyValuePair.h
+//  CollapsibleExample
+//
+//  Created by Lizzy Randall on 6/10/15.
+//  Copyright (c) 2015 Lizzy Randall. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@interface MHKeyValuePair : NSObject
```

Comment:
I went with a simple keyvalue pair since it could have duplicate keys. 

The filter can have then the following example:
How long have you been in America? 6 months
How long have you been in America? 2+ years

Otherwise it would be an array of NSDictionaries, with one key value pair each. I kind of wish they have a simpler singleton class like NSPair..

Do you have any suggestion on a better class to use? If it wouldn't allow for duplicates there would need to be multiple values ex: How long have you been in America? 2+ years, 6 months, etc for the values from that key.